jenkinsapi_tests/unittests/test_job.py

```diff
@@ -27,12 +27,32 @@ def fake_get_data(cls, url, tree=None):  # pylint: disable=unused-argument
         return configs.JOB_DATA
 
     monkeypatch.setattr(JenkinsBase, 'get_data', fake_get_data)
-
     new_job = Job('http://halob:8080/job/foo/', 'foo', jenkins)
 
     return new_job
 
 
+@pytest.fixture(scope='function')
+def jenkins_with_diff_baseurl(monkeypatch):
```

Comment:
This looks similar to some code above.  Could it be moved to `conftest.py`?

Reply:
Moved the jenkins obj creation to conftest.py